packages/component/src/Localization/da-DK.js

```diff
@@ -1,23 +1,83 @@
+function botSaidSomething(avatarInitials, text, timestamp) {
+  return `Bot ${ avatarInitials } sagde, ${ text }, ${ xMinutesAgo(timestamp) }`;
+}
+
+function userSaidSomething(avatarInitials, text, timestamp) {
+  return `Bruger ${ avatarInitials } sagde, ${ text }, ${ xMinutesAgo(timestamp) }`;
+}
+
+function xMinutesAgo(dateStr) {
+  const date = new Date(dateStr);
+  const dateTime = date.getTime();
+
+  if (isNaN(dateTime)) {
+    return dateStr;
+  }
+
+  const now = Date.now();
+  const deltaInMs = now - dateTime;
+  const deltaInMinutes = Math.floor(deltaInMs / 60000);
+  const deltaInHours = Math.floor(deltaInMs / 3600000);
+
+  if (deltaInMinutes < 1) {
+    return 'Lige nu';
+  } else if (deltaInMinutes === 1) {
+    return 'Et minut siden';
+  } else if (deltaInHours < 1) {
+    return `${ deltaInMinutes } minutter siden`;
+  } else if (deltaInHours === 1) {
+    return `En time siden`;
+  } else if (deltaInHours < 5) {
+    return `${ deltaInHours } timer siden`;
+  } else if (deltaInHours <= 24) {
+    return `Idag`;
+  } else if (deltaInHours <= 48) {
+    return `Igår`;
+  } else if (window.Intl) {
+    return new Intl.DateTimeFormat('da-DK').format(date);
+  } else {
+    return date.toLocaleString('da-DK', {
+      day: '2-digit',
+      hour: '2-digit',
+      hour12: false,
+      minute: '2-digit',
+      month: '2-digit',
+      year: 'numeric',
+    });
+  }
+}
+
+
 export default {
-  // FAILED_CONNECTION_NOTIFICATION: '',
+  FAILED_CONNECTION_NOTIFICATION: 'Kunne ikke tilslutte',
   // Do not localize {Retry}; it is a placeholder for "Retry". English translation should be, "Send failed. Retry."
   SEND_FAILED_KEY: 'ikke sendt, {Retry}.',
```

Comment:
Should this be `Ikke`? (Capital I)

Reply:
No it should definitely be 'Kunne ikke tilslutte'. Thanks for the reviews!